packages/mdc-chips/README.md

```diff
@@ -257,8 +257,8 @@ Mixin | Description
 `mdc-chip-trailing-icon-color($color, $opacity, $hover-opacity, $focus-opacity)` | Customizes the color of a trailing icon in a chip, optionally customizes regular/hover/focus opacities
 `mdc-chip-leading-icon-size($size)` | Customizes the size of a leading icon in a chip
 `mdc-chip-trailing-icon-size($size)` | Customizes the size of a trailing icon in a chip
-`mdc-chip-leading-icon-margin($top, $right, $bottom, $left)` | Customizes the margin of a leading icon in a chip
-`mdc-chip-trailing-icon-margin($top, $right, $bottom, $left)` | Customizes the margin of a trailing icon in a chip
+`mdc-chip-leading-icon-margin($right-margin, $left-margin)` | Customizes the margin of a leading icon in a chip
```

Comment:
For some reason I'm not able to comment on previous comment thread. Thanks GitHub!

So are you saying having `mdc-chip-leading-icon-margin($left-margin, $right-margin)` padding is throwing lint errors?

Reply:
Nope. Just that the order of usage in the CSS would then be different than the order in the params. Let me know if you'd like me to change it.

Reply:
I think it's ok to have param order different than CSS style property order. Keeping `left` as first option makes the API intuitive IMO.

Reply:
Just following up on this one. Let me know if you need any help getting it resolved.